python/paddle/fluid/contrib/slim/quantization/quantization_mkldnn_pass.py

```diff
@@ -547,7 +547,21 @@ def _restore_var(self, name, array):
         tensor = self._scope.find_var(name).get_tensor()
         tensor.set(array, self._place)
 
+    def _update_activations(self, graph):
+        for op in graph.all_op_nodes():
+            if op.name() in self._conv_ops and not op.op().has_attr(
+                    "fuse_activation"):
+                activation = ""
+                if op.op().has_attr("fuse_relu") and op.op().attr("fuse_relu"):
+                    activation = "relu"
+                elif op.op().has_attr("fuse_brelu") and op.op().attr(
+                        "fuse_brelu"):
+                    activation = "brelu"
```

Comment:
Is it ok to set activation to `brelu`? I saw in paddle, we now use `relu6` instead of `brelu` now. Because if we set `brelu`, we need to set `brelu_threshold` too.

Reply:
True, done.